u8timeseries/utils/timeseries_generation.py

```diff
@@ -215,11 +227,11 @@ def us_holiday_timeseries(length: int = 10,
     Returns
     -------
     TimeSeries
-        Binary TimeSeries for US holidays.
+        Binary TimeSeries for country's holidays.
```

Comment:
I would add a comment saying that the method returns (at the moment) a daily time series.